CODEOWNERS

```
@@ -0,0 +1,5 @@
+# Reflect current maintainers allowed to approve pull requests that touch the .github
+# and test/e2e repos. This is the TriggerMesh engineering team
+
+/.github/	@triggermesh/engineering
+/test/e2e/	@triggermesh/engineering
```

This subdirectory is sufficient I think.
We probably don't want to have to validate if someone sends a pull request that touches e.g. the Dependabot config.

```suggestion
/.github/workflows/	@triggermesh/engineering
/test/e2e/		@triggermesh/engineering
```

---

Wasn't too sure since encompassing `/.github` would include the configs stored inside as well.

---

What I mean is that `.github/` doesn't contain only Actions related files.
The Dependabot config, and potentially other files such issue templates, are also in this directory. For those, we don't need to approve workflow runs (because they don't access repository secrets).